packages/plugin-prisma/tests/index.test.ts

```diff
@@ -65,6 +65,54 @@ describe('prisma', () => {
     `);
   });
 
+  it('skips fields based on @skip and @include directives', async () => {
```

Comment:
sorry to keep asking for more changes one at a time, but would also be nice to check the inverted condition, so that if the conditions is reversed the selection includes the expected relations

Reply:
no worries! Just pushed one more commit

Reply:
thanks!